spec/helpers/seeds_helper_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe SeedsHelper do
+  describe "display_seed_description" do
+    it "no description" do
```

Comment:
The convention for RSpec test descriptions is that they should form a sentence describing the object under test (in this case, the seed description). Here that would be something like

```
  describe "seed description" do
    it "is missing" do
...
    it "is less than 130 characters long" do
```
etc.

This results in more readable output when the test fails. Currently you get

```
Failures:

  1) SeedsHelper display_seed_description no description
```

as opposed to

```
Failures:

  1) SeedsHelper seed description is missing
```